consensus/raft/log_op.go

```diff
@@ -61,6 +62,15 @@ func (op *LogOp) ApplyTo(cstate consensus.State) (consensus.State, error) {
 			op.Cid,
 			&struct{}{},
 			nil)
+	case LogOpUpdate:
+		err = state.Rm(op.Cid.ToPin().Cid)
+		if err != nil {
+			goto ROLLBACK
+		}
+		err = state.Add(op.Cid.ToPin())
```

Comment:
Removing before adding has no benefits.. adds just replace. My impression is that this is equivalent to LogOpPin.